docker-connector-product/README.md

```diff
@@ -1,15 +1,16 @@
 # Docker Connector
 
-YOUR DESCRIPTION GOES HERE
+Create, start, stop, remove Docker containers with the Axon Ivy's Docker Connector directly from your business processes.
```

Comment:
@andreasbalsiger be warned, new product are being enrolled into the market which you may later want to revise :)